service-api/module/Application/src/KBV/KBVServiceInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\KBV;
+
+interface KBVServiceInterface
+{
+    /**
+     * @return array{
+     *  question: string,
+     *  prompts: string[],
+     *  answer: string
+     * }[]
+     */
+    public function getKBVQuestions(): array;
```

Comment:
This method isn't used externally so can probably be removed from the interface and made `private` if kept in implementations.

Appreciate this method was part of the interface that was on the ticket, but if the return value isn't useful then there's no point keeping it.